ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -342,12 +358,13 @@ Rectangle {
             }
         }
 
-        if ((event.key === Qt.Key_C) && (event.modifiers & Qt.ControlModifier)) {
+        if (((event.key === Qt.Key_C) || (event.key === Qt.Key_X)) && (event.modifiers & Qt.ControlModifier)) {
```

Comment:
Better change that to `event.matches(StandardKey.Copy/Paste)`. What you're testing here means that the modifiers _contain_ the CTRL key, so this would match also if I pressed `Ctrl+Shift+C`

Reply:
Done